src/types.rs

```diff
@@ -348,14 +418,99 @@ impl<T> Line<T>
     }
 }
 
+/// A LineString, which is an ordered collection of [`Point`s](struct.Point.html).
+///
+/// Create a LineString by calling it directly:
+///
+/// ```
+/// use geo::{LineString, Point};
+/// let line = LineString(vec![Point::new(0., 0.), Point::new(10., 0.)]);
+/// ```
+///
+/// Converting a `Vec` of `Point`-like things:
+///
+/// ```
+/// # use geo::{LineString, Point};
+/// let line: LineString<f32> = vec![(0., 0.), (10., 0.)].into();
+/// ```
+///
+/// Or `collect`ing from a Point iterator
+///
+/// ```
+/// # use geo::{LineString, Point};
+/// let mut points = vec![Point::new(0., 0.), Point::new(10., 0.)];
+/// let line: LineString<f32> = points.into_iter().collect();
+/// ```
+///
+/// You can iterate over the points in the `LineString`
+///
+/// ```
+/// use geo::{LineString, Point};
+/// let line = LineString(vec![Point::new(0., 0.), Point::new(10., 0.)]);
+/// for point in line {
+///     println!("Point x = {}, y = {}", point.x(), point.y());
+/// }
+/// ```
+///
```

Comment:
With these PR I just wanted to mimic the standard constructors, so that it would easier to create those objects. Hence I didn't do anything about validity. (But the project would have decide *what* to do w.r.t. validity anyway ;) )